src/style.css

```diff
@@ -0,0 +1,161 @@
+@font-face {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
you don't need @font-face here, you don't import font
just add font-family to html tag